_data/glossary/cla.yaml

```diff
@@ -0,0 +1,8 @@
+id: cla
+name: CLA (Contributor License Agreement)
+tags:
+- community
+short-description: |
+  Terms under which the contributor grants a license to the open source project for their contributions.
```

Comment:
Terms under which a contributor grants a license to an open source project for their contributions.